Source/SpatialGDK/Private/EngineClasses/SpatialNetBitWriter.cpp

```diff
@@ -18,14 +18,14 @@ void FSpatialNetBitWriter::SerializeObjectRef(UnrealObjectRef& ObjectRef)
 	*this << ObjectRef.Entity;
 	*this << ObjectRef.Offset;
 
-	uint8 HasPath = !ObjectRef.Path.empty();
+	uint8 HasPath = !ObjectRef.Path.IsSet();
 	SerializeBits(&HasPath, 1);
 	if (HasPath)
 	{
-		*this << *ObjectRef.Path.data();
+		*this << ObjectRef.Path.GetValue();
 	}
 
-	uint8 HasOuter = !ObjectRef.Outer.empty();
+	uint8 HasOuter = !ObjectRef.Outer.IsSet();
```

Comment:
This is opposite of what it should be (`uint8 HasOuter = ObjectRef.Outer.IsSet();`)